charts/schema-loading/templates/batchjob-schema-loading.yaml

```diff
@@ -55,6 +56,18 @@ spec:
         - "-j"
         - "{{ .Values.schemaLoading.contactPoints }}"
         {{- end }}
+        {{- else }}
+        - "--config"
+        - "database.properties"
```

Comment:
If `schemaLoading.databaseProperties` is not null, this chart create `ConfigMap` and mount it as `database.properties.tmpl`.
When the container starts, it runs the `dockerize` command and `database.properties` will be created.
This chart specifies the above generated `database.properties` as a value of the `--config` flag.